### ISSUE TITLE
session support 

### ISSUE THREAD

**Issue by xiake999** [gowww/app#3]
hello @arthurwhite ,Consider adding session support to the Context?

**Comment by xthezealot** [comment]
Definitely! This will also reach my needs in the next weeks so it's on my TODO list.

**Comment by xiake999** [comment]
Looking forward to!

**Comment by xthezealot** [comment]
Otherwise, [JSON Web Tokens](https://jwt.io) with [dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) could be a better alternative.

**Comment by xiake999** [comment]
@arthurwhite Is JWT in the plan?

**Comment by xthezealot** [comment]
Sorry but my previous statement was stupid... Using JWT just overcomplicates things here.

I've added [gowww/encrypt](https://github.com/gowww/encrypt) which is now integrated in gowww/app.
So we already have AES-128 encrypted cookies. Complete session support is coming.

> Is JWT in the plan?

No, [dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) does the job and its usage is too specific to integrate it in gowww/app.

**Comment by xiake999** [comment]
@arthurwhite Thank you. The problem has been solved
